App_LocalResources/View.ascx.resx

```diff
@@ -1,471 +1,483 @@
-  </data>
-  <data name="CheckHiddenSystemFilesReason.Text" xml:space="preserve">
-    <value>Hackers may upload rootkits into the website, they marked them as system file or hidden in file system, then you can not see these files in file explorer.</value>
-  </data>
-  <data name="CheckHiddenSystemFilesSuccess.Text" xml:space="preserve">
-    <value>There are no files marked as system file or hidden in the website folder.</value>
-  </data>
-  <data name="Check.Text" xml:space="preserve">
-    <value>Check</value>
-  </data>
+&lt;p&gt;Added file(s):&lt;br/&gt;&lt;b&gt;[Path]&lt;/b&gt;&lt;/p&gt;&lt;p&gt;Thanks,&lt;/p&gt;&lt;p&gt;[ModuleName] [ModuleVersion]&lt;/p&gt;</value>
+  </data>
+  <data name="PotentialDangerousFile.Text" xml:space="preserve">
+    <value>A potentially dangerous file has been added to your website</value>
+  </data>
+  <data name="RestrictFileMail_Subject.Text" xml:space="preserve">
+    <value>Potentially dangerous file added to your website</value>
+  </data>
+  <data name="CheckHttpModulesFailure.Text" xml:space="preserve">
+    <value>The Security Analyzer Http Module haven't initialized in application, please check web.config file to see whether the http module config correctly.</value>
+  </data>
+  <data name="CheckHttpModulesName.Text" xml:space="preserve">
+    <value>Check Security Analyzer Http Module</value>
+  </data>
+  <data name="CheckHttpModulesReason.Text" xml:space="preserve">
+    <value>The Security Analyzer Http Module will monitor the website folder and all subfolders, and notify the host manager(s) of any potentially suspicious files created in the site.</value>
+  </data>
+  <data name="CheckHttpModulesSuccess.Text" xml:space="preserve">
+    <value>The Security Analyzer Http Module is installed.</value>
+  </data>
+  <data name="CheckHiddenSystemFilesFailure.Text" xml:space="preserve">
+    <value>There are files marked as system file or hidden in the website folder.</value>
+  </data>
+  <data name="CheckHiddenSystemFilesName.Text" xml:space="preserve">
+    <value>Check Hidden Files</value>
+  </data>
+  <data name="CheckHiddenSystemFilesReason.Text" xml:space="preserve">
+    <value>Hackers may upload rootkits into the website, they marked them as system file or hidden in file system, then you can not see these files in file explorer.</value>
+  </data>
+  <data name="CheckHiddenSystemFilesSuccess.Text" xml:space="preserve">
+    <value>There are no files marked as system file or hidden in the website folder.</value>
+  </data>
+  <data name="Check.Text" xml:space="preserve">
+    <value>Check</value>
+  </data>
+  <data name="CheckDnnVersionFailure.Text" xml:space="preserve">
+    <value>There are major known security vulnerabilities in all versions of DNN Platform up until DNN 9.x, please upgrade to the latest version of DNN Platform.  Once on DNN 9.x, uninstall the Security Analyzer module, it is replaced by the Security Persona Bar extension in DNN 9.x.</value>

```

Comment:
I would propose a modification to this text.

There are multiple major known security vulnerabilities if all versions of DNN Platform prior to 9.x, please upgrade to the latest version of DNN Platform.  Warning, when upgrading the Security Analyzer module should be uninstalled prior to an upgrade to prevent a possible incremental upgrade issue.